.travis.yml

```diff
@@ -0,0 +1,22 @@
+before_script:
+  - sh -e /etc/init.d/xvfb start
+
+language: python
+python:
+  - 2.7
+
+before_install:
+  - "git submodule update --init"
+
+install:
+  - "pip install -r https://raw.github.com/Mozilla/webqa-infrastructure-requirements/master/requirements.txt"
```

Comment:
I believe the webqa-infra requirements are needed

add
`- "git submodule update --init"`